src/components/twilio-voice-dialer/routes.js

```diff
@@ -55,9 +60,29 @@ router.post('/twiml', (req, res) => {
     .send(twiml.toString());
 });
 
-router.post('/conference-events', (req, res) => {
-  // Handle conference events
-  console.log(req.body);
+router.post('/conference-events', async (req, res) => {
+  const { ConferenceSid, StatusCallbackEvent } = req.body;
+
+  if (StatusCallbackEvent === 'participant-join') {
+    const participants = await client.conferences(ConferenceSid).participants.list();
+
+    // Send the conference sid and the other participants' callSids to any client leg.
+    // The sids will be used to update the participant resource such as putting it on hold.
+    participants.forEach((currentParticipant) => {
+      if (currentParticipant.label.split('-')[0] === 'client') {
```

Comment:
just curious, what happens to the 'number' participant?

Reply:
Since number participants cannot call rest APIs, we ignore them here.